### PR TITLE
bug: Queues performance

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: queues/services.py [hunk 1]
```diff
@@ -7,17 +7,16 @@
...
-    data = get(request, QUEUES_URL)
-    if convert_to_options:
-        converted = []
-
-        for queue in data.json().get("queues"):
-            converted.append(Option(queue.get("id"), queue.get("name"), description=queue.get("team").get("name")))
-
-        return converted
+def get_queues(request, disable_pagination=True, page=1, convert_to_options=False):
+    data = get(request, QUEUES_URL + "?page=" + str(page) + "&disable_pagination=" + str(disable_pagination)).json()
```
Comment (jfaracik) [inline queues/services.py:11]: If you used an F string you wouldn't need to convert

File: templates/queues/manage.html [hunk 2]
```diff
@@ -50,6 +50,7 @@ <h1 class="govuk-heading-l">
 				{% endfor %}
 			</tbody>
 		</table>
+		{% include "includes/pagination.html" with data=queues %}
```
Comment (jfaracik) [inline templates/queues/manage.html:53]: {% pagination with data=queues %}

Comment (BenAndrew-Kainos) [reply templates/queues/manage.html]: done

File: queues/services.py (hunk 1 above)
Comment (BenAndrew-Kainos) [reply queues/services.py]: done

**Pull request merged**